age/keysource_test.go

```diff
@@ -24,73 +24,104 @@ func TestMasterKeysFromRecipientsEmpty(t *testing.T) {
 func TestMasterKeyFromRecipientWithLeadingAndTrailingSpaces(t *testing.T) {
 	assert := assert.New(t)
 
-	key, err := MasterKeyFromRecipient("  age1yt3tfqlfrwdwx0z0ynwplcr6qxcxfaqycuprpmy89nr83ltx74tqdpszlw  ")
+	commaSeparatedRecipients := "  age1yt3tfqlfrwdwx0z0ynwplcr6qxcxfaqycuprpmy89nr83ltx74tqdpszlw  ,  age1tmaae3ld5vpevmsh5yacsauzx8jetg300mpvc4ugp5zr5l6ssq9sla97ep  "
```

Comment:
@cedi Let's keep (or add) one test that verifies that `MasterKeyFromRecipient()` works with just one key, as a similar safe guard for compatibility. After that is added I'd say this is ready to be merged!

Reply:
good idea. I'm on it

Reply:
I just realised you actually made the mistake you warned about [here](https://github.com/mozilla/sops/pull/966#discussion_r830294838) and why you suggested making `MasterKeyFromRecipient()` private. It's not `MasterKeyFromRecipient()` but `MasterKeysFromRecipients()` 😄 

The naming on those functions is really unfortunate. I think making the "singular" version private was the right call!